packages/ngx-agenda/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@acpaas-ui/ngx-agenda",
+  "version": "0.0.1",
+  "dependencies": {
+    "hammerjs": "~2.0.8",
+    "@acpaas-ui/js-date-utils": "~1.0.1"
+  },
+  "peerDependencies": {
+    "@angular/core": "~8.2.14",
```

Comment:
shouldn't this be >=8.0.0 to ensure compatibility with all angular 8 and 9 versions (same remark in other package.json files)